ci/blackbox/integration_suite_test.go

```diff
@@ -90,12 +92,20 @@ func TestSuite(t *testing.T) {
 			awsSession := session.New(&aws.Config{
 				Region: aws.String(suiteData.RdsBrokerConfig.RDSConfig.Region)},
 			)
+
+			err := CleanUpTestDatabaseInstances(suiteData.RdsBrokerConfig.RDSConfig.DBPrefix, awsSession)
```

Comment:
The rationale here was delete databases which tests didn't tidy up themselves, for example because the test failed before it could get to that point. I agree with your subsequent comment though.